build/debs/kubernetes-cni.preinst.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+mkdir -p /opt/cni/bin
```

Comment:
This change seems unrelated?

Reply:
couldn't test locally without it as it's a dependency, so semi related, but will do as new pr